### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by tarajan17** [suyashm002/watsappp_bot#7]
I'm from Russia Moscow, I'm very happy to be your potential customer, but I'd like to know if there's an opportunity for my language?? I just don't speak English well. And because of that, you have to go to the translator all the time 
taraska14882281@gmail.com

Best regards to the developer, user.

**Comment by suyashm002** [comment]
Yes we can make it for Russian language also . You can customize as
you wish .

**Comment by tarajan17** [comment]
How do I do that?

**Comment by suyashm002** [comment]
I'll customize it for you give me your requirements and we shall start
working on it
